requirements/common_constraints.txt

```diff
@@ -25,7 +25,7 @@ Django<5.0
 elasticsearch<7.14.0
 
 # django-simple-history>3.0.0 adds indexing and causes a lot of migrations to be affected
-3.0.0
```

Comment:
It seems like maybe this was mis-formatted?  Should the comment above still exist if this is being removed?

Reply:
Ok, sounds good.  That's upstream weirdness and we can look at it over there.